### DISCUSSION TITLE
[BUG] Repository names & description automatically disappears.

### DISCUSSION THREAD

**Discussion by MidlajYH** [community/community#34560] [Mobile] [answered]
https://user-images.githubusercontent.com/85442225/192989259-f0e39b5a-b183-4e1e-953c-d8437cbd74fc.mp4

My Device: Poco X3 Pro
Android Version: 12

Using Github Mobile Latest Beta 1.83.0-beta (10087)

**Comment by Iron-Ham** [comment]
@MidlajYH thank you for reporting this! I've created an issue so we can track this internally and have it fixed.

**Comment by aconsuegra** [comment, marked as answer]
@MidlajYH thanks for the report! Can you reproduce the same issue if you set the app in light theme?

**Reply by MidlajYH** [reply]
@aconsuegra Hi, sorry for the late reply

https://user-images.githubusercontent.com/85442225/193304470-6677c1b4-46ac-45ca-8f39-aa69e5aa4fd6.mp4

It's same in both dark and light theme.

**Reply by aconsuegra** [reply]
Thanks! We have been able to fix the bug and will be shipping a new beta version today with the fix. Thanks again for the report!
